beta-src/.eslintrc.json

```diff
@@ -30,7 +30,8 @@
     "jest/prefer-to-have-length": "warn",
     "jest/valid-expect": "error",
     "react/jsx-filename-extension": ["warn", { "extensions": [".tsx"] }],
-    "no-use-before-define": "off"
+    "no-use-before-define": "off",
+    "no-shadow": "off"
```

Comment:
i think these are dangerous to disable. we should sync on why you made this change. this disables calling a variable before it's defined and using a variable out of scope. idk the scenario that caused this change, but the side effects of this are bad. we should look into a possible one off fix, not a global rule change

Reply:
I ran into an error that I had already declared a type/enum in "an upper scope". I hadn't done this but found a discussion mentioning this: https://github.com/SonarSource/SonarTS/issues/906

I opted to disable the rule but later went back and created exports for the type/interface and didn't re-add the rule. I'll remove the rule, I think the error should be gone now.

Reply:
what happened with this? did you forget to remove or add more? can we disable the error locally instead of globally?